src/main/java/thePackmaster/packs/MarisaPack.java

```diff
@@ -35,4 +36,12 @@ public ArrayList<String> getCards() {
         cards.add(EarthlightRay.ID);
         return cards;
     }
+
+    @Override
+    public PackSummary getPackSummary() {
+        PackSummary pSum = new PackSummary(4, 1, 3, 4,3);
+        pSum.setTags("Exhaust", "Attacks");
```

Comment:
Since there's a specific list of supported tags, it might be easier for people if we make constants for each one. It wasn't needed with the previous way of defining this since it just needed to have the right entry in ui.json. 

There's already a constant for None (`PackSummaryReader.NONE_TAG`) because of how that was treated specially, so just need to add constants like that for the rest:
```
"Strength"
"Exhaust"
"Orbs"
"Stances"
"Discard"
"Debuffs"
"Attacks"
"Tokens"
"Powers"
```

We could also let people just put any string for a tag. The reason I set it up to only allow a specific set of tags is because the pack summary is most useful if it highlights things that are shared between packs. Things specific to one pack can be noted in the pack description.